specification/loadtestservice/LoadTestService/routes.tsp

```diff
@@ -0,0 +1,438 @@
+import "@azure-tools/typespec-azure-core";
+import "@typespec/rest";
+import "@typespec/http";
+import "@typespec/openapi";
+import "./models.tsp";
+import "@azure-tools/typespec-autorest";
```

Comment:
Why import "@azure-tools/typespec-autorest"? I think this is for your to config in `tspconfig.yaml`'s `emit` to generate swagger. Why will code use this lib?

Reply:
This is used because without this we were not able to use @example decorator to attach an example to one of our operation, as it has a custom OperationId. Please refer to stop operation in line 236 of routes.tsp file.

Reply:
Why do you have a custom operation id though? Either you don’t want to rename it from the previous swagger and then you should have the operation name be what is matching. Or you do want to rename then why have operation id?

Reply:
Also it should still resolve the examples automatically.

Reply:
Yes, we have an already published GA version which we are onboarding to TypeSpec, which already has SDK version released for them. We don't want to rename it. It is a custom action stop but its operation ID is LoadTestRun_StopTestRun instead of LoadTestRun_Stop which is generated by the typespec.

Reply:
Meanwhile I agree that example tag is not needed even if we use operation tag . Removed its reference and related import statements.